content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/sboms-and-attestations.md

```diff
@@ -0,0 +1,84 @@
+```
+
+Note that after creating an attestation, you can verify it with Cosign's `verify-attestation` subcommand.
+
+```sh
+cosign verify-attestation $IMAGE
+```
+
+`cosign attach`, on the other hand, only attaches an SBOM to an image and uploads it to a registry. In this case the SBOM isn't signed, meaning that Chainguard Enforce won't have any way to verify its authenticity.
+
+```sh
+cosign attach sbom --sbom image.sbom $IMAGE
+```
+
+This will upload the SBOM to the registry as an OCI artifact with a `.sbom` extension.
+
+Be aware that there is also the `cosign sign` command. After running `cosign attach` to attach an SBOM and upload it to a registry, you can then run `cosign sign` to sign the SBOM, and upload the signature to the registry as a separate OCI artifact, this time with the `.sig` extension.
+
+
+## Should I attest or attach?
+
+Attestations are generally seen as being more trustworthy than unsigned SBOMs, because you can identify who signed them and when. However, Chainguard Enforce will not ingest an attestation unless there is an active policy covering it. 
+
+The reason for this is that if Enforce detects that an image has an attached attestation, then it will attempt to verify its signature by checking it against a policy. If it didn't do this, then Enforce would essentially be ingesting attestations without validating them, which defeats their purpose. Although the policy requirement for attestations generated by a `cosign attest` command is an extra step, this added security layer is often worth the peace of mind it brings.
+
+Conversely, Chainguard Enforce will ingest attached SBOMs out of the box, but cannot apply policies against them. With that said, if you can trust with absolute certainty that an unsigned SBOM is accurate (say, if you yourself created and attached it to the image) or if you don't care about policy enforcement, then attaching SBOMs may suit your needs. 
+
+Say you ran a `cosign attach` command to attach a `.sbom` artifact to an image and upload it to a registry. Then you ran `cosign sign` to sign the SBOM, generate a `.sig` artifact, and upload that to the registry. In this scenario, Chainguard Enforce would still see the SBOM as a `.sbom` artifact and not as an attestation (`.att`). Even though you've signed the SBOM, the signature is a separate artifact so Enforce cannot treat it as an attestation, and thus still won't be able to apply policies against it.
+
+The following table presents a high-level comparison of the `cosign attach` and `cosign attest` commands.
+
+|   | `cosign attach` | `cosign attest` |
+|----------|----------|----------|
+| Produces an in-toto attestation (a "signed SBOM") | no  | yes  |
+| SBOM can be ingested immediately | yes  | no  |
+| SBOM requires policy for ingestion | no | yes  |
+| Policies can be applied to SBOM | no | yes |
+| Uploads OCI artifact to registry | yes | yes |
+| OCI artifact extension | `.sbom` | `.att` |
+
+
+## A Note on generating SBOMs
```

Comment:
I would lower case "Note" since the other headers only have title case on the first word.
```suggestion
## A note on generating SBOMs
```